products/openssl.md

```diff
@@ -0,0 +1,57 @@
+---
+title: OpenSSL
+layout: post
+category: framework
+iconSlug: openssl
+permalink: /openssl
+releasePolicyLink: https://www.openssl.org/policies/releasestrat.html
+eolColumn: Supported
+activeSupportColumn: false
+command: openssl version
+releaseDateColumn: true
+sortReleasesBy: releaseCycle
+releases:
+  - releaseCycle: "3.0"
+    eol: 2026-09-07
+    release: 2021-09-07
+    latest: "3.0.3"
+    link: https://www.openssl.org/news/changelog.html#openssl-30
+    lts: true
+    
+  - releaseCycle: "1.1.1"
+    eol: 2023-09-11
+    release: 2018-09-11
+    latest: "1.1.1o"
+    link: https://www.openssl.org/news/changelog.html#openssl-111
+    lts: true
+    
+    
+  - releaseCycle: "1.1.0"
+    eol: 2019-09-11
+    release: 2016-08-26
+    latest: "1.1.0l"
+    link: https://www.openssl.org/news/changelog.html#openssl-110
+    
+  - releaseCycle: "1.0.2"
+    eol: 2019-12-31
+    release: 2015-01-22
+    latest: "1.0.2u"
+    link: https://www.openssl.org/news/changelog.html#openssl-110
```

Comment:
wrong link probably it should be https://www.openssl.org/news/changelog.html#openssl-102